src/plugins/visualizations/public/wizard/dialog_navigation.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { EuiLink, EuiIcon, EuiSpacer, EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React from 'react';
+
+interface DialogNavigationProps {
+  goBack: () => void;
+}
+
+function DialogNavigation(props: DialogNavigationProps) {
+  return (
+    <>
+      <EuiLink data-test-subj="goBackLink" onClick={props.goBack}>
+        <EuiFlexGroup alignItems="center" gutterSize="s" responsive={false}>
+          <EuiFlexItem grow={false}>
+            <EuiIcon type="arrowLeft" />
+          </EuiFlexItem>
+          <EuiFlexItem>
+            {i18n.translate('visualizations.newVisWizard.goBackLink', {
+              defaultMessage: 'Go Back',
```

Comment:
Also according to our [writing guidelines](https://elastic.github.io/eui/#/guidelines/writing), we should avoid title cases.

```suggestion
              defaultMessage: 'Go back',
```